watertap/examples/flowsheets/RO_with_energy_recovery/mc_sweep_params.yaml

```
@@ -0,0 +1,37 @@
+#================================================================
+A_comp:
+  type: NormalSample
+  model_value: m.fs.RO.A_comp
```

Similar in this file
```suggestion
  model_value: fs.RO.A_comp
```

Also, I'm not sure why we wouldn't call this `model_param` or just `param`.

---

`param` works for me, fixed.